### PR TITLE
Fix translate empty tag values

### PR DESCRIPTION
**Pull request by raphaelquati**
Fix for #5063 

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

**Comment by glinton** [comment]
Also, don't forget to sign the  [CLA](https://influxdata.com/community/cla/).

**Comment by raphaelquati** [comment]
I've already sign! Forget to check!

File: plugins/outputs/azure_monitor/azure_monitor.go [hunk 1]
```diff
@@ -359,12 +359,19 @@ func translate(m telegraf.Metric, prefix string) (*azureMonitorMetric, error) {
...
+		var value = tag.Value
+
+		if key == "" {
 			continue
 		}
 
-		dimensionNames = append(dimensionNames, tag.Key)
-		dimensionValues = append(dimensionValues, tag.Value)
+		if value == "" {
+			value = "<empty>"
```
Comment (danielnelson) [inline plugins/outputs/azure_monitor/azure_monitor.go]: Can dimensions be added with the value as the empty string instead of using a placeholder value?

Comment (danielnelson) [reply plugins/outputs/azure_monitor/azure_monitor.go]: If we can't have empty strings at all, I'd like to try this instead:
```diff
diff --git a/plugins/outputs/azure_monitor/azure_monitor.go b/plugins/outputs/azure_monitor/azure_monitor.go
index 5c435ac0..1c933072 100644
--- a/plugins/outputs/azure_monitor/azure_monitor.go
+++ b/plugins/outputs/azure_monitor/azure_monitor.go
@@ -340,6 +340,9 @@ func hashIDWithTagKeysOnly(m telegraf.Metric) uint64 {
        h.Write([]byte(m.Name()))
        h.Write([]byte("\n"))
        for _, tag := range m.TagList() {
+               if tag.Key == "" || tag.Value == "" {
+                       continue
+               }
                h.Write([]byte(tag.Key))
                h.Write([]byte("\n"))
        }
```

Comment (raphaelquati) [reply plugins/outputs/azure_monitor/azure_monitor.go]: Yes. This is better implementation. I've tested here with success.
image

**Pull request merged**